libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -656,4 +664,46 @@ export const buildTigerSpecificFunctions = (
             };
         }
     },
+    getDataSourceSchemata: async (dataSourceId: string) => {
+        return await authApiCall(async (sdk) => {
+            return await sdk.scanModel.getDataSourceSchemata({ dataSourceId }).then((res: AxiosResponse) => {
+                return Promise.resolve(res?.data);
```

Comment:
I don't think you need to wrap the results into the `Promise.resolve` you should be able to just return them, they will be wrapped into a promise implicitly since this happens in an async function (same for all the functions added in this PR). Or am I missing something?